drivers/rotator/wanderer_rotator_lite.cpp

```diff
@@ -0,0 +1,432 @@
+/*
+WandererAstro WandererRotatorLite
+   Copyright (C) 2020 Jasem Mutlaq (mutlaqja@ikarustech.com)
```

Comment:
You must add your name here, not mine :)

Reply:
Ok :)